src/quantum/azext_quantum/_help.py

```diff
@@ -283,3 +283,39 @@
         text: |-
             az quantum workspace show -g MyResourceGroup -w MyWorkspace
 """
+
+helps['quantum workspace update'] = """
+    type: command
+    short-summary: Enable or disable Azure Quantum workspace api keys.
```

Comment:
quantum workspace update should be broader than just enable/disable key.
Maybe something like
short-summary: Update the given (or current) Azure Quantum workspace.